dev/io.openliberty.webcontainer.servlet.6.1.internal_fat/fat/src/io/openliberty/webcontainer/servlet61/fat/tests/Servlet61HTTPServletMappingTest.java

```diff
@@ -0,0 +1,196 @@
+    }
+
+    @Test
+    public void test_mappingMatch_EXACT() throws Exception {
+        String URI = "/MyServlet";
+        String expectedString = "mappingMatch [EXACT] matchValue [MyServlet] pattern [/MyServlet] servletName [MyServlet]";
+
+        LOG.info("====== <test_mappingMatch_EXACT> ======");
+        runTest(URI, expectedString);
+    }
+
+    @Test
+    public void test_mappingMatch_DEFAULT_3() throws Exception {
+        String URI = "//MyServlet/foo";
+        String expectedString = "mappingMatch [DEFAULT] matchValue [] pattern [/] servletName [MyServlet]";
+
+        LOG.info("====== <test_mappingMatch_DEFAULT_3> ======");
+        runTest(URI, expectedString);
+    }
+
+    @Test
+    public void test_mappingMatch_EXTENSION_1() throws Exception {
+        String URI = "/foo.extension";
+        String expectedString = "mappingMatch [EXTENSION] matchValue [foo] pattern [*.extension] servletName [MyServlet]";
+
+        LOG.info("====== <test_mappingMatch_EXTENSION_1> ======");
+        runTest(URI, expectedString);
+    }
+
+    @Test
+    public void test_mappingMatch_EXTENSION_2() throws Exception {
+        String URI = "/bar/foo.extension";
+        String expectedString = "mappingMatch [EXTENSION] matchValue [bar/foo] pattern [*.extension] servletName [MyServlet]";
+
+        LOG.info("====== <test_mappingMatch_EXTENSION_2> ======");
+        runTest(URI, expectedString);
+
+    }
+
+    @Test
+    public void test_mappingMatch_PATH_1() throws Exception {
+        String URI = "/path/foo";
+        String expectedString = "mappingMatch [PATH] matchValue [foo] pattern [/path/*] servletName [MyServlet]";
+
+        LOG.info("====== <test_mappingMatch_PATH_1> ======");
+        runTest(URI, expectedString);
+    }
+
+    @Test
+    public void test_mappingMatch_PATH_2() throws Exception {
+        String URI = "/path/foo/bar";
+        String expectedString = "mappingMatch [PATH] matchValue [foo/bar] pattern [/path/*] servletName [MyServlet]";
+
+        LOG.info("====== <test_mappingMatch_PATH_2> ======");
+        runTest(URI, expectedString);
+    }
+
+    private void runTest(String URI, String stringToSearch) throws Exception {
+        String url = "http://" + server.getHostname() + ":" + server.getHttpDefaultPort() + "/" + TEST_APP_NAME + URI;
+        HttpGet getMethod = new HttpGet(url);
+
+        LOG.info("Sending request ["+ url + "]");
+
+        try (final CloseableHttpClient client = HttpClientBuilder.create().build()) {
+            try (final CloseableHttpResponse response = client.execute(getMethod)) {
+                String responseText = EntityUtils.toString(response.getEntity());
+                LOG.info("\n" + "Response Text: \n[" + responseText + "]");
+
+                if (stringToSearch != null) {
+                    assertTrue("Expected string not found [" + stringToSearch + "]", responseText.contains(stringToSearch));
+                }
+            }
+        }
+    }
+}
```

Comment:
@pmd1nh  does any of these tests, test the same function as:

1) https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCGetMappingSlashStarTest.java
2) https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCGetMappingTest.java

If so I think we can reduce the number of tests here.  Let me know.

Reply:
There are 4 new added examples in Java Doc.  The remaining should have been covered by the FAT 4.0 tests.
Updating to only test these 4.

  URI Path (in quotes)         matchValue      pattern         mappingMatch
 "MyServlet/index.html"     ""              /               DEFAULT
 "MyServlet/foo"            ""              /               DEFAULT
 "/bar/foo.extension"       bar/foo         *.extension     EXTENSION
 "/path/foo/bar"            foo/bar         /path/*         PATH